test/plugins/sqlite.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const expect = require("chai").expect;
+const Msg = require("../../src/models/msg");
+const Helper = require("../../src/helper");
+const MessageStorage = require("../../src/plugins/sqlite.js");
+
+describe("SQLite Message Storage", function() {
+	const expectedPath = path.join(Helper.getHomePath(), "logs", "testUser.sqlite3");
+	let store;
+
+	// Delete database file from previous test run
+	before((done) => {
+		store = new MessageStorage();
+
+		if (fs.existsSync(expectedPath)) {
+			fs.unlink(expectedPath, done);
+		} else {
+			done();
+		}
+	});
+
+	it("should resolve an empty array when disabled", (done) => {
+		store.getMessages(null, null).then((messages) => {
+			expect(messages).to.be.empty;
+			done();
+		});
+	});
```

Comment:
What about using https://github.com/domenic/chai-as-promised?

For example, this test would become:

```js
it("should ...", function () {
	return expect(store.getMessages(null, null)).to.eventually.be.empty;
});
```

Reply:
Doesn't seem worthwhile to reduce 2 lines into one.

Reply:
I'd argue it makes tests easier to comprehend, and it can obviously be used in more than one place, but whatever. We can use `async`/`await` as soon as we drop support for Node v6, post April 2019 :trollface: